src/actions/helpers/index.ts

```diff
@@ -0,0 +1,21 @@
+import { BigNumber, BigNumberish } from "ethers";
+import { Registrar } from "../../contracts/types";
+
+export const validateOwner = async (
```

Comment:
Nit: consider renaming `validateUserOwnsDomain` or `ensureUserOwnsDomain` to make it more clear what this function does.